ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -220,8 +220,8 @@ Sometimes, you just can't get ActiveRecord to do what you want it to. In that ca
 
 1. Read the first 6 sections of the [Rails Guide on Active Record Querying](http://guides.rubyonrails.org/active_record_querying.html) for a more basic overview of query functions. Don't worry too much about batching and `#find_each`.
 2. Read section 20 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects) for a brief look at using `exists?`, `any?` and `many?`.
-3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
-4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).
+3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#grouping) for an understanding of aggregate functions and the calculations you can run on them.
+4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions). 
```

Comment:
Really sorry mate. I should have picked this up before but I don't think we want sections 9 - 12 here. I think this might be another corrected one from the merge because the next line is for section 12. I'm guessing this should be sections 9 - 11

Reply:
Nevermind mate, please can you check the link in the fourth bull point, it's actually pointing to the section 8.
Normally, i think it should be from section 8 to 11.

![Screenshot 2024-01-05 at 10-13-49 Active Record Queries The Odin Project](https://github.com/TheOdinProject/curriculum/assets/109919684/1a8ef3d2-1df7-4433-996f-9ebaa94aa548)

However, we've already added section 8 in the third bull point🤔

Reply:
Yeah I think point 3 should be section 7 and 21 and then point 4 should be 8 - 11